interface/ceed-jit-tools.c

```diff
@@ -140,23 +141,40 @@ int CeedLoadSourceToInitializedBuffer(Ceed ceed,
       // -- Load local "header.h"
       char *next_quote = strchr(first_hash, '"');
       char *next_new_line = strchr(first_hash, '\n');
-      bool is_local_header = is_hash_include && next_quote
-                             && (next_new_line - next_quote > 0);
-      if (is_local_header) {
+      bool is_local_header = is_hash_include && next_quote &&
+                             (next_new_line - next_quote > 0);
+      char *next_left_chevron = strchr(first_hash, '<');
+      bool is_ceed_header = is_hash_include && next_left_chevron &&
+                            (next_new_line - next_left_chevron > 0) &&
+                            !strncmp(next_left_chevron, "<ceed/", 6) &&
+                            strncmp(next_left_chevron, "<ceed/ceed.h>", 13);
```

Comment:
I think it might be a good idea to
a) exclude more files here or
b) include only the path `<ceed/jit-source/*>` and the files `<ceed/types.h>`, `<ceed/ceed-f32.h>` and `<ceed/ceed-f64.h>`?